tpl/inflect/inflect.go

```diff
@@ -53,7 +54,8 @@ func (ns *Namespace) Humanize(in interface{}) (string, error) {
 		return _inflect.Ordinalize(word), nil
 	}
 
-	return _inflect.Humanize(word), nil
+	str := _inflect.Humanize(word)
+	return _inflect.Humanize(strings.ToLower(str)), nil
```

Comment:
I understand the ToLower, but why the double Humanize?

Reply:
In most case `buffalo/reflect` capitalize the first character. At first, I was trying to use `ToUpper` for the first character, and `ToLower` for the rest. 
```go
return str[:1] + strings.ToLower(str[1:]), nil
```
but this would break this test case.
> {ns.Humanize, "óbito", "Óbito"},

So I decide to lowercase the whole string first then use `Humanize ` to capitalize the first character

Reply:
Sure, but why doing it twice, couldn't you just:


```go
return _inflect.Humanize(strings.ToLower(word)), nil
```

Reply:
No,  that would break those case

> {ns.Humanize, "MyCamel", "My camel"},

for which result is `Mycamel` rather than `My camel`

`word` is raw, unprocessed input string, so it should get `Humanize ` before `ToLower`. Because of inconsistency between ` markbates/inflect` and `buffalo/reflect`. Second `Humanize` is more like a workaround to keep `hugo` unchanged.